cmd/builder/internal/builder/main_test.go

```diff
@@ -78,6 +78,7 @@ var (
 		"/otelcol",
 		"/pdata",
 		"/pdata/testdata",
+		"/pdata/pprofile",
```

Comment:
What is this needed for?

Reply:
Without it, the test fails.
https://github.com/open-telemetry/opentelemetry-collector/actions/runs/9597639894/job/26467167308?pr=10401